07-sampling.Rmd

```diff
@@ -794,32 +738,71 @@ In the upcoming Chapter \@ref(confidence-intervals) on confidence intervals, we'
+Observe the centers of all three sampling distributions: they are all roughly centered somewhere between 0.35 and 0.4 (or between 35% and 40%). Recall from earlier that the value of the population proportion $p$ of the $N$ = `r num_balls` balls in the bowl was `r num_red`/`r num_balls` = `r prop_red` = `r percent_red_chr`. We computed this value by performing a virtual census of `bowl`. Let's re-display our sampling distributions from Figure \@ref(fig:comparing-sampling-distributions-1b), but now with a vertical red line marking the true population proportion $p$ of red balls = `r percent_red_chr` in Figure \@ref(fig:comparing-sampling-distributions-3). We see that while there is a certain amount of error in the sample proportions $\widehat{p}$ for all three sampling distributions, on average the $\widehat{p}$ are centered at the true population proportion red $p$.
+
+```{r comparing-sampling-distributions-3, echo=FALSE, fig.cap="Three sampling distributions with population proportion $p$ marked by vertical line.", purl=FALSE}
+p <- bowl %>%
+  summarize(mean(color == "red")) %>%
+  pull()
+samp_distn_compare <- virtual_prop %>%
+  mutate(
+    n = str_c("n = ", n),
+    n = factor(n, levels = c("n = 25", "n = 50", "n = 100"))
+  ) %>%
+  ggplot(aes(x = prop_red)) +
+  geom_histogram(
+    binwidth = 0.05, boundary = 0.4,
+    color = "black", fill = "white"
+  ) +
+  labs(
+    x = expression(paste("Sample proportion ", hat(p))),
+    title = expression(paste(
+      "Sampling distributions of ", hat(p),
+      " based on n = 25, 50, 100."
+    ))
+  ) +
+  facet_wrap(~n) +
+  geom_vline(xintercept = p, col = "red", size = 1)
+
+if (is_latex_output()) {
+  samp_distn_compare +
+    theme(
+      strip.text = element_text(colour = "black"),
+      strip.background = element_rect(fill = "grey93")
+    )
+} else {
+  samp_distn_compare
+}
+```
+
+
+Moreover, observe that when the sample size used is 25 (left histogram) it is somewhat likely to obtain sample proportions of red balls that are less than 0.2 = 20% or greater than 0.55 = 55%; when the sample size is 50 (middle histogram) it is unlikely, although still possible, to get that extreme sample proportions; and when the sample size is 100 (right histogram) we have not observed a single sample proportion that is that extreme. As the sample size $n$ increases from 25 to 50 to 100, \index{sampling distributions!relationship to sample size} the variation of the sampling distribution decreases, and thus the values cluster more and more tightly around the same center between 35% and 40%. We quantified this variation using the standard deviation of our sample proportions in Table \@ref(tab:comparing-n), which we display again as Table \@ref(tab:comparing-n-repeat):
```

Comment:
Maybe three different sentences instead of connecting them with ;? Or turn this into bullet points for even more clarity?

Reply:
Got it. But, it will change once CLT is merged with it.